src/components/AllSpecies/AllSpecies.test.tsx

```diff
@@ -0,0 +1,20 @@
+import AllSpecies from './AllSpecies'
+import { render, screen } from '@testing-library/react'
+import { Tabs, TabPanels } from '@chakra-ui/react'
+import { mockModifiedCharacters } from '../../api/mocks/mockModifiedCharacters'
+
+describe('AllSpecies', () => {
+  test('render multiple kinds of spiceis characters in one list', () => {
+    render(
+      <Tabs>
+        <TabPanels>
+          <AllSpecies characters={mockModifiedCharacters} />
+        </TabPanels>
+      </Tabs>,
+    )
+    const humans = screen.getAllByText(/human/i)
+    expect(humans.length).toBeGreaterThan(0)
+    const droids = screen.getAllByText(/droid/i)
+    expect(droids.length).toBeGreaterThan(0)
```

Comment:
[nitpick] - it's a bit more readable to couple them based on their needs rather than cases like the following;
```suggestion
    // assign
    const humans = screen.getAllByText(/human/i)
    const droids = screen.getAllByText(/droid/i)

    // asserts
    expect(humans.length).toBeGreaterThan(0)
    expect(droids.length).toBeGreaterThan(0)
```